Cargo.toml

```diff
@@ -51,3 +53,18 @@ std = [
   "byteorder/std",
   "ring/std",
 ]
+
+tool = [
+  "clap",
+  "serde",
+  "serde_json",
+]
```

Comment:
I really, really don't like that you have to add this feature. I can see why you had to. I may change it so that these are all default features, since on-devices uses of Manticore already have to set `default-features = false`... Sigh.

Let's leave it as-is for now.

Reply:
ACK.